fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -192,7 +193,7 @@ public Triple apply(final Statement input) {
         }
         servletResponse.addHeader("Vary", "Accept, Range, Accept-Encoding, Accept-Language");
 
-        return Response.ok(rdfStream).build();
+        return ok(rdfStream.withThisContext(rdfStream.transform(new Deskolemizer(idTranslator, null)))).build();
```

Comment:
In that case, I will rebase and maybe tighten up how this works in the HTTP layer and we should be able to move forward.